packages/node/test/node/node-helpers.ts

```diff
@@ -141,6 +141,10 @@ export function CommissioningHelper() {
                     await agent.operationalCredentials.addTrustedRootCertificate({ rootCaCertificate: params.caCert });
                 });
 
+                //const fabrics = node.env.get(FabricManager);
```

Comment:
These and the "await promise" below are just adding comments, should it be removed?

Reply:
Yes left overs ... removed